src/styles/blocks/card-block/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+  }
+
+  @for $starCount from 1 through 4 {
+    .stars__star:nth-child(-n + #{$starCount}) {
+      background-image: url(../images/star-active.svg);
```

Comment:
Use absolute paths for assets like images instead of relative paths. Change url(../images/star-active.svg) to url(/images/star-active.svg)